src/Native/Common/utilcode.h

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+#include "daccess.h"
+
+#define THUMB_CODE 1
+
+template <typename ResultType, typename SourceType>
+inline ResultType DataPointerToThumbCode(SourceType pCode)
```

Comment:
Since `DataPointerToThumbCode/ThumbCodeToDataPointer` is not used in CoreRT, it may be better to just inline the implementation into `PCODEToPINSTR`/``PINSTRToPCODE`.

Reply:
Done